### ISSUE TITLE
About your code in pwlu

### ISSUE THREAD

**Issue by lemon-prog123** [MrGoriay/pwlu-pytorch#1]
`mean = x.detach().mean([0,1,2,-1]) #{TODO}: Possibly split along channel axis
            var = x.detach().var([0,1,2,-1]) #{TODO}: Possibly split along channel axis`

It seems that you forgot to use detach() so that the tensor x can detach from cau graph,since that torch.mean() will use a lot of GPU Memory when you are doing  a trainning section.

**Comment by MrGoriay** [comment]
Thanks for pointing out, implemented your fix and closing.
